src/myframe.h

```diff
@@ -964,6 +966,10 @@ struct MyFrame : wxFrame {
             case A_AUTOEXPORT:
                 sys->cfg->Write(L"autohtmlexport", sys->autohtmlexport = ce.IsChecked());
                 break;
+            case A_JPEG_DEFAULT_IMAGETYPE:
+                sys->defaultimagetype = (ce.IsChecked()) ? wxBITMAP_TYPE_JPEG : wxBITMAP_TYPE_PNG;

```

Comment:
I don't think we want to globally change all images from png to jpeg or vice versa.
Instead, each image should be the type it gets loaded at.. typically diagrams are in png, photos in jpeg etc.
Then, if anyone wants to change it, we can allow it on a per image basis, though frankly I wonder if that is ever needed.